src/status_im2/contexts/quo_preview/settings/accounts.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im2.contexts.quo-preview.settings.accounts
+  (:require [clojure.string :as string]
+            [quo2.components.settings.accounts.view :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as r]
```

Comment:
`[reagent.core :as reagent]`

Reply:
Again, solved in 3d9a6bf0afe3430696defb1e18fa8de2d9a94eaa